FreeRTOS/FreeRTOS
# Update MQTT demo timeout and record length

Description
-----------
Update MQTT plain text demo timeout and record length to handle more packets. 

Test Steps
-----------
All demos run in Jenkins.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
